### ISSUE TITLE
[FEATURE] Google data layer script

### ISSUE THREAD

**Issue by tcdahlberg** [SFDO-Community-Sprints/Summit-Events-App#476]
**Describe the solution you'd like**
Add a simple data layer for Google Tag Manager to each page. 

```
<script>
dataLayer = [{
     'account': ***account_name_if_exists***,
     'recordType': ***recordType_developerName***,
     'eventLabel': ***title_of_event***
}];
</script>
```
*** = to be replaced with data from the event

**Comment by rdblake21** [comment]
Updates to the following, which is embedded on each page:

```           
<script>
                dataLayer = [{
                    'account': '{!IF(!ISBLANK(eventPage.Account__r.Name),eventPage.Account__r.Name, '')}',
                    'audience': '{!IF(!ISBLANK(eventPage.Audience__c),eventPage.Audience__c, '')}',
                    'eventLabel': '{!eventPage.Event_Name__c}{!IF(!ISBLANK(eventInstance.Instance_Title__c),' - ' + eventInstance.Instance_Title__c,'')}',
                    'category': '{!IF(!ISBLANK(eventPage.Filter_Category__c), eventPage.Filter_Category__c, '')}',
                    'sponsor': '{!IF(!ISBLANK(eventPage.Event_Sponsor__c), eventPage.Event_Sponsor__c, '')}',
                    'page': '{!$CurrentPage.Name}'
                }];
            </script>

**Comment by tcdahlberg** [comment]
Included in beta release 0.29.0.1

**Comment by tcdahlberg** [comment]
Included in production release 0.29.0.1
